cli/cmd/create.go

```diff
@@ -64,6 +64,7 @@ func runCreate(
 	ctx := cmd.Context()
 	message := workspace_repo.Message(fmt.Sprintf("Happy %s Create Stack [%s]", util.GetVersion().Version, stackName))
 	err = validate(
+		validateConfigurationIntegirty(ctx, happyClient),
```

Comment:
we should probably put this in the push command too

Reply:
Done.